docs/docs/features/ml-hardware-acceleration.md

```diff
@@ -36,9 +35,16 @@ You do not need to redo any machine learning jobs after enabling hardware accele
 
 #### CUDA
 
-- You must have the official NVIDIA driver installed on the server.
+- The GPU must have compute capability 5.2 or greater.
+- The server must have the official NVIDIA driver installed.
+- The installed driver must be >= 535 (it must support CUDA 12.2).
 - On Linux (except for WSL2), you also need to have [NVIDIA Container Runtime][nvcr] installed.
 
+#### OpenVINO
+
+- The server must have a discrete GPU, i.e. Iris Xe or Arc. Expect issues when attempting to use integrated graphics.
```

Comment:
One or more ML tasks will just fail, see #8226 and #8353. There have been a few attempts to fix it, but fixing one issue has led to it breaking for others.

Reply:
Thanks for pointing me to those issues.  I also notice myself that facial recognition started failing a few weeks ago when using OpenVino, when previously it worked just fine.  I'm using Intel N100 which has integrated graphics.  I had to disable OpenVino in order to get Immich working again.

What I don't understand from this new documentation is, do we expect these new issues to never be fixed, or is this just a temporary thing that we can't use OpenVino anymore?

If it is permanently broken, why does it no longer work?  Did the model get too big or complex for the N100 integrated graphics?  If so, what's your minimum specification?

Reply:
We updated OpenVINO and onnxruntime-openvino a month ago. That fixed two issues relating to external data and dynamic input shapes, but seems to have a major regression in iGPU compatibility.

As for fixing it, this is up to upstream in OpenVINO and/or ONNX Runtime. The prerequisites right now are just because of the current behavior in these libraries. I opened an issue about it and imagine the issue will be fixed at some point, but I don't know when.

Reply:
Since both openvino issues seem to be resolved, would it make sense to remove the statement that you shouldn't use integrated GPUs?

Reply:
They're resolved for now, but iGPU support (particularly for older GPUs) is brittle and can (and has) broken with updates to ONNX Runtime and OpenVINO. I'm fine with officially supporting the more reliable set of hardware and unofficially trying to get it working on older hardware.